aea/registries/base.py

```diff
@@ -30,7 +30,7 @@
 from pathlib import Path
 from typing import Optional, List, Dict, Any, Tuple, cast, Union
 
-from aea.configurations.base import ProtocolId, SkillId, ProtocolConfig, DEFAULT_PROTOCOL_CONFIG_FILE
+from aea.configurations.base import ProtocolId, SkillId, ProtocolConfig, DEFAULT_PROTOCOL_CONFIG_FILE, Address
```

Comment:
so is `Address` still in `configurations.base`? Shouldn't it be in `mail.base`?

Reply:
Yes but first I will make changes based #200 #474 and after I finished these I will change the #410 .